### PR TITLE
Forward port of 7.10.1 release notes to 7.x

### PR DESCRIPTION
**Pull request by andsel**
Forward port of 7.10.1 release notes (#12489)

### CODE REVIEW

File: docs/static/releasenotes.asciidoc [hunk 1]
```diff
@@ -3,6 +3,7 @@
 
 This section summarizes the changes in the following releases:
 
+* <<logstash-7-10-0,Logstash 7.10.1>>
```
Comment (robbavey) [inline docs/static/releasenotes.asciidoc:6]: Should be `7-10-1` on the left side?

File: docs/static/releasenotes.asciidoc [hunk 2]
```diff
@@ -36,6 +37,47 @@ This section summarizes the changes in the following releases:
...
+=== Logstash 7.10.1 Release Notes
+
+==== Notable issues fixed
+
+===== Fix docker image metadata
+Removed non-OCI compliant freeform metadata labels, started using consistent build date for all the docker images and dockerfiles. https://github.com/elastic/logstash/pull/12451[#12451]
+Reintroduces the labels for the ubi8 image only. https://github.com/elastic/logstash/pull/12500[#12500]
+
+===== Exclude file with non-commercial license
+Bundled did_you_mean gem had a CC NonCommercial license in an included file, these files has been removed. https://github.com/elastic/logstash/pull/12460[#12460]
```
Comment (robbavey) [inline docs/static/releasenotes.asciidoc:50]:
```suggestion
Bundled did_you_mean gem had a CC NonCommercial license in an included file, these files have been removed. https://github.com/elastic/logstash/pull/12460[#12460]
```

File: docs/static/releasenotes.asciidoc [hunk 3]
```diff
@@ -36,6 +37,47 @@ This section summarizes the changes in the following releases:
...
 * <<logstash-7-0-0-alpha1,Logstash 7.0.0-alpha1>>
 
+[[logstash-7-10-1]]
+=== Logstash 7.10.1 Release Notes
+
+==== Notable issues fixed
+
+===== Fix docker image metadata
+Removed non-OCI compliant freeform metadata labels, started using consistent build date for all the docker images and dockerfiles. https://github.com/elastic/logstash/pull/12451[#12451]
+Reintroduces the labels for the ubi8 image only. https://github.com/elastic/logstash/pull/12500[#12500]
```
Comment (robbavey) [inline docs/static/releasenotes.asciidoc:47]: Use consistent tense for release note comments

```suggestion
Reintroduced the labels for the ubi8 image only. https://github.com/elastic/logstash/pull/12500[#12500]
```

**Review by robbavey — CHANGES_REQUESTED** [review]

File: docs/static/releasenotes.asciidoc (hunk 1 above)
Comment (andsel) [reply docs/static/releasenotes.asciidoc:6]: cut & paste error :-(

File: docs/static/releasenotes.asciidoc (hunk 3 above)
Comment (andsel) [reply docs/static/releasenotes.asciidoc:47]: Thank's Rob, as Joao is suggesting, the message for this release is the fix of pipeline reloading other than the other https://github.com/elastic/logstash/pull/12489#discussion_r537550747 so probably this part is going to be removed

File: docs/static/releasenotes.asciidoc [hunk 4]
```diff
@@ -36,6 +37,40 @@ This section summarizes the changes in the following releases:
...
+*Azure_event_hubs Input - 1.2.3*
+
+* Fixed missing configuration of `prefetch_count` and `receive_timeout` https://github.com/logstash-plugins/logstash-input-azure_event_hubs/pull/61[#61]
+
+*Http Input - 3.3.6*
+
+* Fixes a regression introduced in **3.1.0**'s migration to the Netty back-end that broke some users'
+browser-based workflows. When a plugin configured to require Basic authentication receives a request that does not
+include authentication, it now appropriately includes an `WWW-Authenticate` header in its `401 Unauthorized` response,
+allowing the browser to collect credentials before retrying the request.
```
Comment (robbavey) [inline docs/static/releasenotes.asciidoc:60]: Missing link to PR

File: docs/static/releasenotes.asciidoc [hunk 5]
```diff
@@ -36,6 +37,40 @@ This section summarizes the changes in the following releases:
...
+
+*Sqs Input - 3.1.3*
+
+* Fix: retry networking errors (with backoff) https://github.com/logstash-plugins/logstash-input-sqs/pull/57[#57]
+
+*Kafka Integration - 10.5.3*
+
+* Fix: set (optional) truststore when endpoint id check disabled https://github.com/logstash-plugins/logstash-integration-kafka/pull/60[#60].
+Since **10.1.0** disabling server host-name verification (`ssl_endpoint_identification_algorithm => ""`) did not allow the (output) plugin to set `ssl_truststore_location => "..."`.
```
Comment (robbavey) [inline docs/static/releasenotes.asciidoc:69]: Nit: Typically the PR link is at the end of the changelog comment

**Pull request merged**